docs/demand/historical/eastern.rst

```diff
@@ -0,0 +1,26 @@
+Eastern
+#######
+The construction of the Eastern Interconnection demand profile is slightly different
+from the its Western counterpart. We still download historical demand data from EIA
```

Comment:
`from the its Western counterpart` -> `from the Western counterpart`?

Reply:
Done